frontend/public/components/custom-resource-definition.tsx

```diff
@@ -107,6 +114,53 @@ const Established: React.FC<{ crd: CustomResourceDefinitionKind }> = ({ crd }) =
   );
 };
 
+const EmptyVersionsMsg: React.FC<{}> = () => <EmptyBox label="CRD Versions" />;
+
+const CRDVersionsHeader = () => [
+  {
+    title: 'Name',
+    sortField: 'name',
+    transforms: [sortable],
+  },
+  {
+    title: 'Served',
+    sortField: 'served',
+    transforms: [sortable],
+  },
+  {
+    title: 'Storage',
+    sortField: 'storage',
+    transforms: [sortable],
+  },
+];
+
+const CRDVersionsRows = ({ componentProps: { data } }) =>
+  _.map(data, (version: CRDVersion) => [
+    {
+      title: version.name,
+    },
+    {
+      title: version.served.toString(),
+    },
+    {
+      title: version.storage.toString(),
+    },
+  ]);
+
+const CRDVersionList: React.FC<CRDVersionProps> = ({ crdVersions }) => {
```

Comment:
naming nit: I'd make it clear it's a table in the name. I'd also call the prop `versions` since it's clear from the component name that we're dealing with CRD versions.

```suggestion
const CRDVersionTable: React.FC<CRDVersionProps> = ({ versions }) => {
```